tests/js/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  extends: ['plugin:testing-library/react'],
+  rules: {
+    'testing-library/render-result-naming-convention': 'off',
+    'testing-library/prefer-screen-queries': 'off',
+  },
```

Comment:
Should these be in `eslint-config-sentry`?

Reply:
Unfortunately, they get overridden by this extend if they are there.

We'll be able to get rid of `testing-library/prefer-screen-queries` line soon, just one more big file to go.

Once we get the extends in overrides feature with the upgraded eslint, we'll be able to delete this whole file.

Reply:
Actually, I can put them there 👍 Just had the wrong order when I was testing it previously.